Rakefile

```diff
@@ -41,7 +41,7 @@ namespace :ci do
     packages = changed_packages
     puts "Running rubocop on: #{packages.join(', ')}"
     packages.each do |package|
-      run_command("cd #{package} && bundle exec rubocop")
+      run_command("cd #{package} && bundle exec rubocop -c ../.rubocop.yml")
```

Comment:
I think this change broke the previous behaviour of checking for config files in the parent directory https://github.com/rubocop-hq/rubocop/pull/8239

Reply:
Although not sure this is true after looking at the code a bit closer